_posts/2017-07-19-NUnit3TestAdapter-3.8.md

```diff
@@ -0,0 +1,9 @@
+NUnit3Testadapter, version 3.8 is released.
+
+Major feature is support for .net core /.net standard. 
```

Comment:
Minor nitpick, I think we should follow the official stylization `.NET Core/.NET Standard`.